### PR TITLE
Update page-layout.md

### PR DESCRIPTION
**Pull request by seneca**
Minor tweaks

### CODE REVIEW

File: packages/nuejs.org/docs/page-layout.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 
 # Page layout
-Global design system standardizes the structure of your web pages so you can use the same HTML markup, but vary your external CSS to achieve wildly different designs.
+A global design system standardizes the structure of your web pages, allowing you to use the same HTML markup while applying different external CSS to achieve a wide range of designs.
 
 
 ## Standardized HTML
-After describing your headers and footers, the default page layout for Markdown content looks like this:
+After describing your headers and footers in `layout.html` file, the default page layout for Markdown content looks like this:
```
Comment (nobkd) [inline packages/nuejs.org/docs/page-layout.md]:
```suggestion
After describing your headers and footers in the `layout.html` file, the default page layout for Markdown content looks like this:
```

File: packages/nuejs.org/docs/page-layout.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
 
 # Page layout
-Global design system standardizes the structure of your web pages so you can use the same HTML markup, but vary your external CSS to achieve wildly different designs.
+A global design system standardizes the structure of your web pages, allowing you to use the same HTML markup while applying different external CSS to achieve a wide range of designs.
```
Comment (nobkd) [inline packages/nuejs.org/docs/page-layout.md]:
```suggestion
A global design system standardizes the structure of your web pages, allowing you to use the same HTML markup, while applying different external CSS, to achieve a wide range of designs.
```

That's how I would read it, feel free to reject

**Review by nobkd — CHANGES_REQUESTED** [review]

File: packages/nuejs.org/docs/page-layout.md (hunk 1 above)
Comment (seneca) [reply packages/nuejs.org/docs/page-layout.md]: Good catch.

File: packages/nuejs.org/docs/page-layout.md (hunk 2 above)
Comment (seneca) [reply packages/nuejs.org/docs/page-layout.md]: 👍

**Review by nobkd — APPROVED** [review]

**Pull request merged**